src/gdash/scripts/_contains/_contains.gml

```diff
@@ -15,9 +15,10 @@ _contains("hello", "ello");
 // => true
 */
 
-var collection = argument[0];
-var target = argument[1]
-var fromIndex = 0;
+var 

```

Comment:
I think there used to be but not anymore. I only changed it because I find it more readable, but I don't mind changing it back, I don't have time for that now though. You can accept it now and leave it, change it back yourself or let me do that tomorrow, all the same to me. And you are correct about the semicolon!